controls/radrichtextbox/features/styles/styles.md

```diff
@@ -164,9 +179,9 @@ Styles of type Character get applied to the currently selected part of the docum
 
 Styles of type Paragraph follow the same logic and are applied to all paragraphs in the selection or the current paragraph. 
 
-For example, the following line will apply the "linkedParagraphStyle" to the current Paragraph and the parts of the text which are selected:
+For example, the code from **Example 5** applies the "linkedParagraphStyle" to the current Paragraph and the parts of the text which are selected.

```

Comment:
Can we avoid the repetition of the word 'example' here? Maybe the sentence could be structured something like this: "In "Example 5" you can see how to apply the "linkedParagraphStyle" to the current Paragraph..."